### PR TITLE
chsh: page update

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
I think there was a stage where these edits were what the pages was, but we changed it for users incase they didn't know where the shells are

File: pages/common/chsh.md [hunk 1]
```diff
@@ -7,9 +7,13 @@
 
 `chsh`
 
+- Change the current user's login shell:
+
+`chsh {{/path/to/shell}}`
```
Comment (blueskyson) [inline pages/common/chsh.md:12]:
```suggestion
`chsh {{path/to/shell}}`
```

File: pages/common/chsh.md [hunk 2]
```diff
@@ -7,9 +7,13 @@
...
+
 - Change the login shell for a given user to Zsh:
 
-`chsh --shell {{/bin/zsh}} {{username}}`
+`chsh --shell {{/path/to/shell}} {{username}}`
```
Comment (blueskyson) [inline pages/common/chsh.md:16]:
```suggestion
`chsh --shell {{path/to/zsh}} {{username}}`
```

Comment (EmilySeville7cfg) [reply pages/common/chsh.md]: I've just removed leading `/` for both paths.

File: pages/common/chsh.md [hunk 3]
```diff
@@ -7,9 +7,13 @@
 
 `chsh`
 
+- Change the current user's login shell:
+
+`chsh {{path/to/shell}}`
+
 - Change the login shell for a given user to Zsh:
```
Comment (navarroaxel) [inline pages/common/chsh.md:14]: But this example description is specific for `zsh`.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**